basemaps/localization.md

```diff
@@ -66,133 +116,97 @@ Extending our Switzerland example with exonym and endonym from other languages:
+
+The following set of rules is used:
+
+- Show local names only if they use a different script than the target language
+- If the target language is not available, fallback to name:en if the local script is not Latin
+- Hide text in scripts that cannot be rendered correctly by MapLibre, such as Khmer or Bengali
+
+### Example: Milano
+
+For a map localized to English, we only use `name:en = Milan` since the local `name = Milano` uses the Latin script which is used in English as well. The label would be:
+
+```
+Milan
+```
+
+For a map localized to Greek, we would use `name:el = Μιλάνο` in the first line and since that is a different script from Latin, we would also include the local `name = Milano` in the second line:
+
+```
+Μιλάνο
+Milano
+```
 
 ## Positioned glyph font `pmap:pgf:name:*` values
 
 Protomaps adds additional names for a small set of language scripts, currently just the `Devanagari` script used for Hindi (`name:hi` and `pmap:pgf:name:hi`) and related languages.
 
 Rendering text in web browsers works for almost all languages and scripts and feels like magic. However, specialized map renderers like MapLibre have to reimplement text rendering and text layout which is complicated when text needs to be curved along linear map features instead of placed only horizontally or vertically. MapLibre normally assumes a one-to-one mapping between glyphs and Unicode codepoints that also exist in MapLibre font files (aka "font stacks") to accomplish the layout for a large but limited number of scripts. Plugins have been developed to extend MapLibre for **right-to-left** scripts like Arabic and Hebrew, and MapLibre has built-in support for **CJK scripts** like Chinese, Japanese, and Korean.
 
-To facilitate Protomap's support of additional, non-supported scripts in MapLibre (like the Devanagari script used by the Hindi language), Protomaps exports names with "positioned glphys" so MapLibre can use codepoints as indices of positioned glyphs in an additional custom "font stack". While the raw `pmap:pgf:name:*` values look like giberish when inspecting the raw values, they render correctly in MapLibre to the end user.
+To facilitate Protomap's support of additional, non-supported scripts in MapLibre (like the Devanagari script used by the Hindi language), Protomaps exports names with "positioned glphys" so MapLibre can use codepoints as indices of positioned glyphs in an additional custom "font stack". While the raw `pmap:pgf:name:*` values look like gibberish when inspecting the raw values, they render correctly in MapLibre to the end user.
 
 See more:
 
 - [Traditional MapLibre Text Rendering](https://oliverwipfli.ch/about-text-rendering-in-maplibre-2023-10-17/)
 - [Devanagari Positioned Glyph Fonts](https://oliverwipfli.ch/devanagari-in-the-protomaps-basemap-with-a-positioned-glyph-font-for-maplibre-2024-06-30/)
 
-## Styling localized name
```

Comment:
While it's ok to take out this placeholder section... we're tilting towards using Protomaps as a platform solution instead of a modular system of tile schema, styles, and data archives. Ideally we offer some tips on how to work with the raw tile data, too?

Reply:
You're right, I think this section under the `basemaps` directory concerns the combination of style + tileset.

I think `Localization` concerns the style, because that is the API surface that developers interact with - generating a GL style for a given language. 

I do think there ought to be one section in the Basemaps directory for the schema with no style opinions - probably fleshed out https://docs.protomaps.com/basemaps/layers (issue #1)? That can mention localized name tags, and link out to this Localization page that has more prose?